Detections/CarbonBlack/CriticalThreatDetected.yaml

```diff
@@ -4,7 +4,7 @@ description: |
   'This creates an incident in the event a critical threat was identified on a Carbon Black managed endpoint.'
 severity: Medium
 requiredDataConnectors:
-  - connectorId: VMwareCarbonBlack
```

Comment:
Please remove this change, this is not custom.  We have a connector in the GitHub that is moving into the UX - https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/VMware%20Carbon%20Black/VMwareCarbonBlack_API_FunctionApp.json